### ISSUE TITLE
New component: Trip.js - ProductTour

### ISSUE THREAD

**Issue by testing138** [primefaces-extensions/primefaces-extensions.github.com#780]
Hi, 

It would be very great, if you can add the following component into PFE:

Here is one example, I guess there are also other JS available.
http://eragonj.github.io/Trip.js/

I would need this component to implement a Guide in my webpage, like you maybe know from intercom. 
Please see as reference this video: https://www.youtube.com/watch?v=TyWefNd1Tlc

For me it´s also important 
- Information can be coming also from database (so I can have a loop around that)
- Content can be also in HTML: <h:outputText value="Text Text" escape="false"/>

**Properties**
- modal 
- currentTab (integer)
- titel (String), but you can also define it in <f:facet name="header">
- startOnPageLoad
- DelayPageLoad
- labelNext
- labelPrevious
- labelEnd
Please see also: http://eragonj.github.io/Trip.js/documentations/configuration/

**Events:**
- Click next page
- Start tour
- Close tour
- Stop

<p:repeat>
<pe:tourGuide>

     <f:facet name="header">
        List of Cars
    </f:facet>

     <h:outputText value="Text Text" />

    <f:facet name="footer">
        List of Cars
    </f:facet>
</pe:tourGuide>
</p:repeat>

Many thanks

**Comment by tandraschko** [comment]
don't get me wrong but you are longtime user and people really often invest time in the forum with you. Maybe a good point to give something back and create the compnent by yourself and provide it here opensource?

**Comment by testing138** [comment]
agreed. if I would have the knowledge to create a component, I would do it. But I have no knowledge to do this.

**Comment by tandraschko** [comment]
thats a non-arg ;) We all started without knowledge. There enough sample components available here to dig in step by step

**Comment by commander-hp** [comment]
Oh yes! This component is awesome. Please integrate this in Primefaces.
